tilt-setup/deploy_tilt_env.sh

```diff
@@ -38,19 +39,38 @@ apply_bm_hosts "${NAMESPACE}"
 # remove bmo, so that is deployed and monitored by Tilt
 kubectl delete deployments.apps -n "${IRONIC_NAMESPACE}" baremetal-operator-controller-manager
 
-pushd "${BMOPATH}"
 # shellcheck disable=SC2155
 # shellcheck disable=SC1001
 export IRONIC_SECRET_NAME=$(kubectl get secrets  -n "${IRONIC_NAMESPACE}" -oname | grep ironic-credentials | cut -f2 -d\/)
 # shellcheck disable=SC2155
 # shellcheck disable=SC1001
 export IRONICINSPECTOR_SECRET_NAME=$(kubectl get secrets  -n "${IRONIC_NAMESPACE}" -oname | grep "ironic-inspector-credentials" | cut -f2 -d\/)
+popd
+
+pushd "${SCRIPTDIR}"
+# Relative path from SCRIPTDIR to BMOPATH
+REL_PATH_TO_BMO="$(realpath --relative-to="${SCRIPTDIR}" "${BMOPATH}")"
+
+# Create overlay
+mkdir -p config/overlays/tilt/
+cat <<EOF > config/overlays/tilt/kustomization.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - ../../../${REL_PATH_TO_BMO}/config/namespace
+  - ../../../${REL_PATH_TO_BMO}/config/default
+
+components:
+  - ../../../${REL_PATH_TO_BMO}/config/components/tls
```

Comment:
nit: Keep indent the same throughout
```suggestion
- ../../../${REL_PATH_TO_BMO}/config/namespace
- ../../../${REL_PATH_TO_BMO}/config/default

components:
- ../../../${REL_PATH_TO_BMO}/config/components/tls
```